docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -1,15 +1,227 @@
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+// TEST
+<1> call to the <<ilm-put-lifecycle,put lifecycle API>> endpoint to create
+    a new policy named "datastream_policy"
+<2> policy definition sub-object
+<3> the hot phase defined in the "phases" section. Optional `min_age` field
+    not defined -- defaults to `0ms`
+<4> rollover action definition
+<5> delete phase begins after 90 days
+<6> delete action definition
+
+
+Here we created the policy called `datastream_policy` which rolls over
+the index being written to after it reaches 50 gigabytes, or it is 30
+days old. The rollover will occur when either of these conditions is true.
+The index will be deleted 90 days after it is rolled over.
+
+=== Applying a policy to our index
+
+There are <<set-up-lifecycle-policy,a few ways>> to associate a
+policy to an index. Since we wish specific settings to be applied to
+the new index created from Rollover, we will set the policy via
+index templates.
+
+
+[source,js]
+-----------------------
+PUT _template/datastream_template
+{
+  "index_patterns": ["datastream-*"],                 <1>
+  "settings": {
+    "number_of_shards": 1,
+    "number_of_replicas": 1,
+    "index.lifecycle.name": "datastream_policy",      <2>
+    "index.lifecycle.rollover_alias": "datastream"    <3>
+  }
+}
+-----------------------
+// CONSOLE
+// TEST[continued]
+<1> match all indices starting with "datastream-". These will include all
+    newly created indices from actions like rollover
+<2> the name of the lifecycle policy managing the index
+<3> the alias to use for the rollover action, since it is defined in the policy.
+
+The above index template introduces a few new settings specific to {ILM}.
+The first being `index.lifecycle.name`. This setting will configure
+the "datastream_policy" to the index applying this template. This means
+that all newly created indices prefixed "datastream-" will be managed by
+our policy. The other setting used here is `index.lifecycle.rollover_alias`.
+This setting is required when using a policy containing the rollover
+action and specifies which alias to rollover on behalf of this index.
+The intention here is that the rollover alias is also defined on the index.
+
+To begin, we will want to bootstrap our first index to write to.
+
+
+[source,js]
+-----------------------
+PUT datastream-000001
+{
+  "aliases": {
+    "datastream": {
+      "is_write_index": true
+    }
+  }
+}
+-----------------------
+// CONSOLE
+// TEST[continued]
+
+When creating our index, we have to consider a few important configurations
+that tie our index and our policy together correctly. We need to make sure
+that our index name matches our index template pattern of "datastream-*",
+which it does. We are using the rollover action in our policy, which
+requires that our index name ends with a number. In our case, we used
+`000001`. This is important so that Rollover can increment this number when
```

Comment:
Can you make `Rollover` here a link to the Rollover documentation?

Reply:
will merge in the policy definition PR before this one so the link exists 👍